lms/static/sass/course/modules/_calculator.scss

```diff
@@ -15,20 +15,19 @@ div.calc-main {
 
   .calc {
     @include transition(background-color $tmg-f2 ease-in-out 0s);
-    background: url("../images/calc-icon.png") $black-t3 no-repeat center;
+    background: url("../images/calc-icon.png") $black-t1 no-repeat center;
```

Comment:
This is very much out of scope for this work to just style/build the UI for toggling student notes. I've changed the value here just to sync up the background/presentation of this element with the notes-related work.

I'd imagine this could be part of a separate "Font Awesome all the glyphs!" story.